topics/genome-annotation/tutorials/funannotate/tutorial.md

```diff
@@ -0,0 +1,478 @@
+
+# Integrating structural and functional annotation
+
+Now we have a structural annotation, and the results of both **EggNOG Mapper** and **InterProScan**. Each one is in a separate file, so we will now combine all this data into a single file that will contain both the structural *and* the functional annotation. This will be the final output of our annotation pipeline, ready to be submitted to the NCBI reference database.
+
+> ### {% icon hands_on %} Hands-on
+>
+> 1. {% tool [Funannotate functional](toolshed.g2.bx.psu.edu/repos/iuc/funannotate_annotate/funannotate_annotate/1.8.9+galaxy2) %} with the following parameters:
+>    - *"Input format"*: `GenBank (from 'Funannotate predict annotation' tool)`
+>        - {% icon param-file %} *"Genome annotation in genbank format"*: `annotation (genbank)` (output of **Funannotate predict annotation** {% icon tool %})
+>    - *"Funannotate database"*: select the latest version available
+>    - {% icon param-file %} *"NCBI submission template file"*: `template.sbt` (the file downloaded from NCBI website, or leave empty if you didn't generate it)
+>    - {% icon param-file %} *"Eggnog-mapper annotations file"*: `annotations` (output of **eggNOG Mapper** {% icon tool %})
+>    - {% icon param-file %} *"InterProScan5 XML file"*: `InterProScan XML` (output of **InterProScan** {% icon tool %})
+>    - {% icon param-file %} *"BUSCO models"*: `mucorales (orthodb 10)`
+>    - *"Strain name"*: `muc1`
+>    - *"locus_tag from NCBI to rename GFF gene models with"*: `MMUCEDO_` (we consider NCBI has assigned us this `locus_tag`)
+>    - *"Which outputs should be generated"*: Select all
+>
+{: .hands_on}
+
+This tool produces several output dataset, in particular:
+
+- The full structural and functional annotation in Genbank and GFF3 format
+- NCBI tbl and sqn (`NCBI Sequin genome`) formats: these files can be used for submission to NCBI
+- The CDS, transcript and protein sequences of all the genes predicted by Funannotate
+- Some statistics and reports
+
+If you display the GFF3 output, you will notice that the functional information, including gene names, is now stored in this file.
+
+# Visualisation with a genome browser
+
+With Galaxy, you can visualize the annotation you have generated using JBrowse. This allows you to navigate along the chromosomes of the genome and see the structure of each predicted gene. We also add an RNASeq track, using the BAM file created with **RNA STAR** {% icon tool %}.
+
+> ### {% icon hands_on %} Hands-on
+>
+> 1. {% tool [JBrowse](toolshed.g2.bx.psu.edu/repos/iuc/jbrowse/jbrowse/1.16.11+galaxy1) %} with the following parameters:
+>    - *"Reference genome to display"*: `Use a genome from history`
+>        - {% icon param-file %} *"Select the reference genome"*: `genome` (output of **Sort assembly** {% icon tool %})
+>    - In *"Track Group"*:
+>        - {% icon param-repeat %} *"Insert Track Group"*
```

Comment:
are there any interproscan tracks that would be interesting here?

Reply:
Not really, you can get a gff3, but the coordinates are on each protein sequences instead of the genome. There could be some conversion maybe (but too late for next week ahah)

Reply:
https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/gff3_rebase/gff3.rebase/1.2 I don't remember writing that very well, but apparently even a checkbox for 'interpro specific modifications'. (But yeah no worries, we can merge as is and file that for later.

Reply:
:+1: writing it in my todo list for later!